voucher/src/main/res/values/styles.xml

```diff
@@ -10,25 +10,38 @@
 
     <style name="AdyenCheckout.Voucher" />
 
-    <style name="AdyenCheckout.Voucher.Description.Bacs" parent="AdyenCheckout.TextAppearance.Primary">
+    <!-- This style is deprecated and will be removed in future releases. Please override
+         AdyenCheckout.SimpleVoucher.Description instead. -->
+    <style name="AdyenCheckout.Voucher.Description.Bacs" parent="AdyenCheckout.SimpleVoucher.Description">
+        <item name="android:text">@string/checkout_voucher_introduction_bacs</item>
+    </style>
+
+    <style name="AdyenCheckout.SimpleVoucher.Description" parent="AdyenCheckout.TextAppearance.Primary">
         <item name="android:layout_marginStart">@dimen/standard_double_margin</item>
         <item name="android:layout_marginEnd">@dimen/standard_double_margin</item>
         <item name="android:layout_marginTop">@dimen/standard_margin</item>
-        <item name="android:text">@string/checkout_voucher_introduction_bacs</item>
         <item name="android:gravity">center_horizontal</item>
     </style>
 
-    <style name="AdyenCheckout.Voucher.Description.Boleto" parent="AdyenCheckout.TextAppearance">
+    <!-- This style is deprecated and will be removed in future releases. Please override
+         AdyenCheckout.FullVoucher.Description instead. -->
+    <style name="AdyenCheckout.Voucher.Description.Boleto" parent="AdyenCheckout.FullVoucher.Description">
+        <item name="android:textSize">12sp</item>
+        <item name="android:text">@string/checkout_voucher_introduction</item>
+    </style>
+
+    <style name="AdyenCheckout.FullVoucher.Description" parent="AdyenCheckout.TextAppearance">
```

Comment:
I like the new separation between simple voucher styles and full voucher styles, but we should fix the naming.
If you go now to the example app and try to override the style from there, you will get an error:
![Screenshot 2024-01-24 at 17 33 43](https://github.com/Adyen/adyen-android/assets/9079915/e1a2355a-43bf-4648-aea7-553860a2879a)

The reason for this is that the style `AdyenCheckout.FullVoucher` is not declared anywhere. That's why you can see an empty style `<style name="AdyenCheckout.Voucher" />` at the top of this file (and similarly in all other styles files).

So we can declare 2 new styles: `AdyenCheckout.SimpleVoucher` and `AdyenCheckout.FullVoucher` but since we already have `AdyenCheckout.Voucher` I will suggest using `AdyenCheckout.Voucher.Simple` and `AdyenCheckout.Voucher.Full`.

WDYT?

Reply:
Good catch, I agree with your suggested changes.